spec/models/item_spec.rb

```diff
@@ -12,14 +12,25 @@
     it { should validate_presence_of :description }
     it { should validate_presence_of :unit_price }
   end
-  
+
   describe 'class methods' do
     before :each do
       @merchant = Merchant.create!(name: "Little Shop of Horrors")
       @item_1 = @merchant.items.create!(name: 'Audrey II', description: 'Large, man-eating plant', unit_price: '100000000', enabled: true)
       @item_2 = @merchant.items.create!(name: 'Bouquet of roses', description: '12 red roses', unit_price: '1900', enabled: true)
       @item_3 = @merchant.items.create!(name: 'Orchid', description: 'Purple, 3 inches', unit_price: '2700', enabled: false)
       @item_4 = @merchant.items.create!(name: 'Echevaria', description: 'Peacock varietal', unit_price: '3100', enabled: true)
+      @item_5 = @merchant.items.create!(name: 'Silver Ring', description: 'Jewelery', unit_price: '2700', enabled: false)
+      @invoice_1 = @item_1.invoices.create!(status: 1, created_at: "2012-03-06 14:54:15 UTC")
+      @invoice_4 = @item_4.invoices.create!(status: 1, created_at: "2012-03-05 14:54:15 UTC")
+      @invoice_2 = @item_2.invoices.create!(status: 1, created_at: "2012-03-07 00:54:24 UTC")
+      @invoice_3 = @item_3.invoices.create!(status: 1, created_at: "2012-03-08 14:54:15 UTC")
+      @invoice_5 = @item_5.invoices.create!(status: 1, created_at: "2012-03-08 14:54:15 UTC")
+      @invoice_item_1 = InvoiceItem.create!(item_id: @item_1.id, invoice_id: @invoice_1.id, status: 1)
+      @invoice_item_2 = InvoiceItem.create!(item_id: @item_2.id, invoice_id: @invoice_2.id, status: 1)
+      @invoice_item_3 = InvoiceItem.create!(item_id: @item_3.id, invoice_id: @invoice_3.id, status: 1)
+      @invoice_item_4 = InvoiceItem.create!(item_id: @item_4.id, invoice_id: @invoice_4.id, status: 1)
+      @invoice_item_5 = InvoiceItem.create!(item_id: @item_5.id, invoice_id: @invoice_5.id, status: 0)
```

Comment:
Oh boy -- we prob should have coordinated testing here, since we're going to have a giant merge conflict with my next branch... 😬 ☠️ ahhh well. I can deal with renaming them on my side.

Reply:
Sorry! Didn't realize. We could remove the before each & put them in individual tests if it's easier?

Reply:
@Sidarth20 nothing to be sorry about -- totally both spaced on this. Leave yours as is. I'm just going to update my tests once I merge this in.